### DISCUSSION TITLE
Concrete differences between Evaluation and Standalone

### DISCUSSION THREAD

**Discussion by jnikodemus** [Security-Onion-Solutions/securityonion#4501] [General] [answered]
The documentation just says that Evaluation is ideal for classroom or small lab environments and "...Standalone is more ready for production usage.". Are there any special, noticeable differences?

**Comment by manuelplascencia** [comment]
I noticed on the SO youtube channel from 2020 conference video it goes into that check it. 

https://youtu.be/SAubxi13K-Q

Thank you,
Manuel Plascencia
Director of Technology & Information Services
Whittier City School District

(562)789-3261
***@***.***
Follow us @WhittierCitySD on FB,  
and @WhittierCity on Twitter
#WCSDTeamKids
San Gabriel Valley Tech Exchange - (Subscribe)
CETPA Region Secretary

**Comment by manuelplascencia** [comment, marked as answer]
While I was looking up Syslog info,  I also found some addition info in here, that might be useful to you.

 https://docs.securityonion.net/en/2.3/ingest.html?highlight=syslog

**Comment by jnikodemus** [comment]
Thanks for your answers, that helps me a lot!

**Comment by dougburks** [comment]
The Architecture section of the documentation has more information and includes diagrams for both Evaluation and Standalone:
https://docs.securityonion.net/en/2.3/architecture.html
